README.md

```diff
@@ -127,19 +127,18 @@ A peer can have the following properties:
 
 Emitted when a new wifi peer connection is discovered.
 
-### var ev = sync.start(peer)
+### var ev = sync.replicate(target)
 
-`peer` is an object with properties `host`, `port`, and `name`.
+`peer` is an object with properties `host`, `port`, and `name` **or** an object
```

Comment:
```suggestion
`target` is an object with properties `host`, `port`, and `name` **or** an object
```

Reply:
I've been renaming it to peers, what do you think about that?

Reply:
Sounds good!